simplq/package.json

```diff
@@ -31,7 +31,7 @@
     "build": "react-scripts build",
     "test": "react-scripts test",
     "eject": "react-scripts eject",
-    "lint": "eslint src --ext .js,.jsx --fix-dry-run",
+    "lint": "eslint src --ext .js,.jsx --fix",
```

Comment:
Do we need these filters by extension? Otherwise, should we add json too?

same for the lint-staged one.

Reply:
Remove fix-dry-run from lintstaged too, line 79

Reply:
It didn't run on all files when I tried using the glob syntax (src/**/**), that's why I used this.